doc/deployment_guide.md

```diff
@@ -0,0 +1,797 @@
+
+- use the `./kc` CLI tool or
+- use the Keymaster WebUI at `http://localhost:4226` or
+- use the Keymaster REST OpenAPI at `http://localhost:4226/api/v1/`
+
+The MDIP [Integration Guide](integration_guide.md) focuses on Keymaster deployments, Keymaster OpenAPI interface, and integration of MDIP wallet functionality with 3rd party applications.
+
+### MDIP Components
+
+![MDIP-components](https://github.com/user-attachments/assets/c88a33f0-aa61-48b9-a62a-ca956f1038d9)
+
+All the MDIP components illustrated above are composed together using Docker Compose in the official ['kc' v.0.3-beta repository](https://github.com/KeychainMDIP/kc/releases/tag/v0.3-beta). The repository provides a simple [`start-node`](https://github.com/KeychainMDIP/kc/blob/v0.3-beta/start-node) script that will compile, dockerize, and launch all required MDIP components. The convenience of this script will be preserved as we explore configuration and deployment options below.
+
+Note that each MDIP components can also be manually launched using native nodeJS commands. The Dockerfile for each component will reveal the particular shell command being executed within the Docker environment. Using the [Dockerfile.gatekeeper](https://github.com/KeychainMDIP/kc/blob/v0.3-beta/Dockerfile.gatekeeper) as an example, we see the command used to launch the Gatekeeper API:
+`CMD ["node", "server/src/gatekeeper-api.js"]`
+
+While perhaps not optimal for a production environment, developement environments can be deployed with all the required components on a single moderately-sized server. The statistics below demonstrate a typical development environment:
+
+![MDIP-resources](https://github.com/user-attachments/assets/7d63d4dc-fa51-4254-b9ad-94b91d98f78d)
+
+- kc-gatekeeper-1 is the [MDIP Gatekeeper](https://keychain.org/docs/mdip/scheme) API process.
+- kc-keymaster-1 is a [Keymaster API](https://keychain.org/docs/keychain/library/api/) to manage the server's MDIP identity.
+- kc-hyperswarm-1 is the document distribution protocol built on [Hyperswarm](https://github.com/holepunchto/hyperswarm).
+- kc-tftc-mediator-1 is the MDIP Mediator for [Feathercoin Testnet](https://feathercoin.com/)
+- kc-tbtc-mediator-1 is the MDIP Mediator for [Bitcoin Testnet](https://github.com/bitcoin/)
+- kc-tftc-node is a generic Feathercoin Testnet node
+- kc-tbtc-node is a generic [Bitcoin Testnet4](https://github.com/bitcoin/bitcoin/pull/29775) node
+
+## Systems Requirements
+
+The resource estimates and recommendations below are early estimates and are still under development.
+
+MDIP operations begin when the MDIP Gatekeeper receives a request from a Keymaster client. From the perspective of a Keymaster (MDIP client), all Gatekeepers are *the same* in the sense that they expose the same MDIP interfaces (REST OpenAPI API and WebUI). MDIP Gatekeepers nodes do not have any visibility in the keys held securely by each MDIP Keymaster Client.
```

Comment:
This section is great content, but not really applicable to the header. Unless objected, I will move to another section, likely "What is an MDIP Node", but subject to change as I continue to review the doc and find other content to move.